docs/contributing/ASPNetCore-Debugging.md

```diff
@@ -1,25 +1,29 @@
-5. You should see the generated packages in the `aspnetcore\artifacts\packages\Debug\NonShipping` directory. The packages should end with `x.0.0-dev.nupkg` where `x` is the current .NET version.
-6. Open `razor-tooling/NuGet.config` and add the local package sources:
-- `<add key="ASPNETCORE_SHIPPING" value="<PATH_TO_ASPNET_CORE_REPO>\artifacts\packages\Debug\Shipping\" />`
-- `<add key="ASPNETCORE_NONSHIPPING" value="<PATH_TO_ASPNET_CORE_REPO>\artifacts\packages\Debug\NonShipping\" />`
+1. `./restore.cmd`
+1. Make the desired changes in `dotnet/aspnetcore`.
+1. `./eng/build.cmd -pack`. The `-pack` option causes the creation of NuGet packages.
+1. You should see the generated packages in the `aspnetcore\artifacts\packages\Debug\NonShipping` directory. The packages should end with `x.0.0-dev.nupkg` where `x` is the current .NET version.
+1. Open `razor-tooling/NuGet.config` and add the local package sources:
+
+   - `<add key="ASPNETCORE_SHIPPING" value="<PATH_TO_ASPNET_CORE_REPO>\artifacts\packages\Debug\Shipping\" />`
+   - `<add key="ASPNETCORE_NONSHIPPING" value="<PATH_TO_ASPNET_CORE_REPO>\artifacts\packages\Debug\NonShipping\" />`
+
+1. Open `razor-tooling/eng/Versions.props` and note the version for `MicrosoftCodeAnalysisRazorPackageVersion`. Ex. `5.0.0-rc.1.20380.7`.
+1. Do a find in `Versions.props` for the version in step 7 and replace with `x.0.0-dev`.
+1. Get the assembly version of the `aspnetcore` packages.
+   1. Assembly version can be found by openning the `.dll` in `ILSpy`
+       1. Ex. `~/.nuget\packages\microsoft.aspnetcore.razor.language\6.0.0-dev\lib\netstandard2.0.nuget\packages\microsoft.aspnetcore.razor.language\{VERSION}.0.0-dev\lib\netstandard2.0\Microsoft.AspNetCore.Razor.Language.dll`
+       2. This is likely going to be the dev version `42.42.42.42`
+1. Update the `OldVersionUpperBound` and `NewVersion`, of the Razor assemblies in `src\Razor\src\Microsoft.VisualStudio.RazorExtension\AssemblyBindingRedirects.cs` with the assembly version from the step above.
 
-7. Open `razor-tooling/eng/Versions.props` and note the version for `MicrosoftCodeAnalysisRazorPackageVersion`. Ex. `5.0.0-rc.1.20380.7`.
-8. Do a find in `Versions.props` for the version in step 7 and replace with `x.0.0-dev`.
-9. Get the assembly version of the `aspnetcore` packages.
-   1.  Assembly version can be found by openning the `.dll` in `ILSpy`
-       1.  Ex. `~/.nuget\packages\microsoft.aspnetcore.razor.language\6.0.0-dev\lib\netstandard2.0.nuget\packages\microsoft.aspnetcore.razor.language\{VERSION}.0.0-dev\lib\netstandard2.0\Microsoft.AspNetCore.Razor.Language.dll`
-       2.  This is likely going to be the dev version `42.42.42.42`
-10. Update the `OldVersionUpperBound` and `NewVersion`, of the Razor assemblies in `src\Razor\src\Microsoft.VisualStudio.RazorExtension\AssemblyBindingRedirects.cs` with the assembly version from the step above.
+## Notes
 
-## Notes:
```

Comment:
Punctuation at the end of headings is not recommended (I suspect it has something to do with navigable headings.